src/wallabyTest.ts

```diff
@@ -17,3 +17,4 @@ TestBed.initTestEnvironment(
     BrowserDynamicTestingModule,
     platformBrowserDynamicTesting()
 );
+TestBed.initTestEnvironment = () => {};
```

Comment:
Type '() => void' is not assignable to type '(ngModule: Type<any> | Type<any>[], platform: PlatformRef, aotSummaries?: () => any[]) => TestBed'.
  Type 'void' is not assignable to type 'TestBed'.
(method) TestBed.initTestEnvironment(ngModule: Type<any> | Type<any>[], platform: PlatformRef, aotSummaries?: () => any[]): TestBed
 Initialize the environment for testing with a compiler factory, a PlatformRef, and an
     * angular module. These are common to every test in the suite.
     *
     * This may only be called once, to set up the common providers for the current test
     * suite on the current platform. If you absolutely need to change the providers,
     * first use `resetTestEnvironment`.
     *
     * Test modules and platforms for individual platforms are available from
     * '@angular/<platform_name>/testing'.
     *
     * @experimental